modules/ossm-threescale-webassembly-module-minimal-working-configuration.adoc

```diff
@@ -39,49 +40,28 @@ spec:
       - no_body
     services:
     - id: '2555417834780'
-      token: service_token
       authorities:
       - "*"
       credentials:
+        user_key:
+          - query_string:
+              keys:
+                - <user_key>
+          - header:
+              keys:
+                - <user_key>
         app_id:
         - header:
```

Comment:
![image](https://user-images.githubusercontent.com/87991133/214696557-b06efb55-beee-486b-a22d-5422a76e50c0.png)

Please move these right and change order of "header" and "query_string" so its consistent